__test__/components/__snapshots__/indexPage.test.js.snap

```diff
@@ -1,3832 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Why was this snapshot deleted?

Reply:
I removed the snapshot earlier because it was failing on Travis. I wrote more tests to cover the lines

Reply:
Not a good way of doing things fam... at least not without calling my attention to it. Let's at least understand why it's failing on travis.